### ISSUE TITLE
Named Formatter breaks format strings with parens

### ISSUE THREAD

**Issue by darraghjones** [scottrippey/SmartFormat.NET#56]
This unit test  fails from version 1.3.

```
[TestFixture]
class Tests
{
    [TestCase(true, "yes (probably)")]
    [TestCase(false, "no (possibly)")]
    public void TestWithParens(bool value, string expected)
    {
        Assert.AreEqual(expected, Smart.Format("{value:yes (probably)|no (possibly)}", new {value }));
    }
}
```

Expected string length 13 but was 0. Strings differ at index 0.

  Expected: "no (possibly)"

  But was:  <string.Empty>

**Comment by scottrippey** [comment]
Will be solved by #65

**Comment by axunonb** [comment]
Will be fixed in version 1.7.x coming up shortly.

**Comment by axunonb** [comment]
Updated version committed, where test with the issues mentioned have passed: https://github.com/scottrippey/SmartFormat.NET/commit/46bbae26b6c34582887b60fc0488cd134514fde7
